build/Dockerfile.base

```diff
@@ -70,8 +70,7 @@ RUN env -u VERSION GOOS=windows make ecr-credential-provider
 # Build CNI plugins
 WORKDIR /build/windows-machine-config-operator/containernetworking-plugins/
 COPY containernetworking-plugins/ .
-ENV CGO_ENABLED=0
```

Comment:
Do we need the update here 

https://github.com/openshift/windows-machine-config-operator/blob/master/build/Dockerfile#L74